### DISCUSSION TITLE
I fail to sign in Radicale service via internal network penetration.

### DISCUSSION THREAD

**Discussion by Jifeng1984** [Kozea/Radicale#1575] [Q&A client]
I attempted to set up a server using PeanutHull for internal network penetration and installed a Radicale service on it. We can now access the server through a web service and reach the Radicale interface. However, when entering the password, an error message appears. Does anyone know what might be causing this issue? Or are there alternative solutions for internal network penetration? Thanks!

Based on the image shown, I am able to reach the login screen. However, once the correct username and password are entered, errors start occurring.

![11111](https://github.com/user-attachments/assets/4313e83a-bb28-4b65-92f7-4fcb1f741632)
![222222](https://github.com/user-attachments/assets/f658e017-7404-45a2-81e4-c176dbb62066)

**Comment by pbiering** [comment]
check server log and potentially also brower log what happend
